templates/zerver/help/send-a-group-of-people-a-private-message.md

```diff
@@ -1,75 +1,64 @@
 # Private Group Messages
-## There are three ways to send a private message to a group of people:
-### First Way:
-To send a new private group message,
-first click on the `New private message`
-button located at the bottom of your screen.
+You can send a private message by using the compose box, the group PM list,
+or private messages tab.
+### Sending a private message through the Compose box
 
-![New private message](/static/images/help/private-message.png)
+1. Click on the **New private message** button located at the bottom of your
+screen. This will make the **compose box** appear at the bottom center of your
+screen.
 
-The following box will be displayed:
+    ![New private message](/static/images/help/private-message.png)
 
-![New stream message](/static/images/help/private-box.png)
-
-In the **one or more people...** field,
+2. In the **one or more people...** field of the **compose box**,
 input the usernames or email addresses of the users
 you want to send a private group message to.
-Each username or email address should be separated by a comma (`,`).
+Each username or email address should be separated by a comma (**,**).
 When you begin typing an email address or username,
 autocomplete suggestions will appear for individuals.
 The list of users in this field creates the group
 to which the private message will be sent.
 
-If you attempt to input a nonexistent username or email address,
-an error message will appear.
+    ![New stream message](/static/images/help/private-box.png)
+
+3. Type your message to that group of users in the
+**Compose your message here...** field. You can now send your private message.
+
+!!! warn ""
+    If you attempt to input a nonexistent username or email address,
+    an error message will appear.
 
-### Second Way:
-If you have already private messaged a group of people,
+### Sending a private message through the Group PMs list
+1. If you have already private messaged a group of people,
 their names will appear under the **Group PMs** heading
 at the bottom of the user list on the right side of your screen.
 
-![Group PMs](/static/images/help/group-pms.png)
+    ![Group PMs](/static/images/help/group-pms.png)
 
-Clicking on one of the groups will bring up a messaging box.
+2. Clicking on one of the groups will bring up the **compose box**.
```

Comment:
I think this should be written in imperative style, "Click on one of the groups", just like you do in the third step.